src/ProgressOnderwijsUtils/Data/DbColumnDefinition.cs

```diff
@@ -27,14 +27,8 @@ public static ColumnDefinition[] GetFromReader([NotNull] IDataRecord reader)
             => Enumerable.Range(0, reader.FieldCount).Select(fI => new ColumnDefinition(reader.GetFieldType(fI), reader.GetName(fI))).ToArray();
 
         [NotNull]
-        public static ColumnDefinition[] GetFromTable([NotNull] SqlConnection sqlconn, string tableName)
-        {
-            using (var cmd = sqlconn.CreateCommand()) {
-                cmd.CommandText = "SET FMTONLY ON; select * from " + tableName + "; SET FMTONLY OFF";
-                using (var fmtReader = cmd.ExecuteReader())
-                    return GetFromReader(fmtReader);
-            }
-        }
+        public static ColumnDefinition[] GetFromTable([NotNull] DatabaseDescription.Table table) 

```

Comment:
Als je de api in temen van iets maakt dat weliswaar de tabel-naam en kolommen-in-db-volgorde bevat, maar niet letterlijk afhankelijk van DatabaseDescription is, dan heb je minder problemen met de temp-tables.

Je zou bijv. in eerste instantie kunnen proberen met tabel-naam en DbColumnMetaData[] ofzo, ook al is dat niet het "mooiste" waarschijnlijk.

Reply:
Je bedoelt dat we dan bij temp. tables die kolommen-array zelf moeten gaan opstellen ipv ze door de database description uit de database te laten lezen?

Reply:
ja; 

alhoewel "moeten" klinkt alsof we dat nog moeten doen - dat doen we al, om de temp tables aan te maken.

Reply:
Zie de CascadeDelete impl voor een voorbeeld.